qiita_db/software.py

```diff
@@ -413,31 +484,37 @@ def exists(cls, command, **kwargs):
         -------
         bool
             Whether if the parameter set exists in the given command
+
+        Raises
+        ------
+        qiita_db.exceptions.QiitaDBError
+            - If there are missing parameters for the given command
+            - If there are extra parameters in `kwargs` than for the given
```

Comment:
This means that there's an error raised if `kwargs` contains parameters not originally defined in command?